app/frontend/controllers/export_controller.js

```diff
@@ -0,0 +1,579 @@
+/* This file is part of Ezra Bible App.
+
+   Copyright (C) 2019 - 2021 Ezra Bible App Development Team <contact@ezrabibleapp.net>
+
+   Ezra Bible App is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 2 of the License, or
+   (at your option) any later version.
+
+   Ezra Bible App is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with Ezra Bible App. See the file LICENSE.
+   If not, see <http://www.gnu.org/licenses/>. */
+
+const marked = require('marked');
+const i18nHelper = require('../helpers/i18n_helper.js');
+const { parseHTML } = require('../helpers/ezra_helper.js');
+
+/**
+ * The ExportController implements the export of certain verses with notes or tags into a Word document.
+ *
+ * @category Controller
+ */
+
+
+var exportFilePath;
+var docx;
+
+module.exports.showSaveDialog = async function (fileTitle) {
+  const dialog = require('electron').remote.dialog;
```

Comment:
Since we do depend on Electron APIs for the export, we should disable the functionality completely on Cordova/Android until we have found an alternative there (I didn't in my previous analysis).

Reply:
That's a good point. I'll try to research. I think it's a really important feature.

Reply:
I think the key is whether you find a way to offer a file download from the context of the Android webview. Maybe there is a Cordova plugin for it? In the worst case I could look into creating one, but then I would have to brush up on my Java skills.

Reply:
I'll disable it for now and leave a comment in the code